tutorials/simulation_exporting.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<p>You seem to use the word 'mixin' on purpose - it is also used further up. What does it mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
It's a class which doesn't interfere with the "main" class inheritance. But as Eirik points out, I haven't done it properly in this PR

Reply:
Thanks for the explanation!